scripts/plugin_list.py

```diff
@@ -0,0 +1,85 @@
+import datetime
```

Comment:
For consistency with the other scripts, let's use `-` as separator here.

Also nitpick of course, but how about naming this script `update-plugins-list.py`?

Reply:
Done!